README.md

```
@@ -0,0 +1,110 @@
+# Install dependencies
+npm install
+
+# Copy environment variables
+cp .env.example .env.local
+```
+
+### Environment Variables
+
+Create a `.env.local` file with:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)